ruby/agoo/Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM ruby:2.5.3
 
 WORKDIR /usr/src/app
 
-COPY Gemfile app.rb ./
+COPY Gemfile app.rb index.html ./
```

Comment:
why adding an `index.html` ?

Reply:
It is an empty file. Since the `/` path returns a blank page it is really a static asset or can be considered as one. Agoo handles static assets directly without involving Ruby so is much faster.

Reply:
I understand, but it is kind of out ouf scope here, I think
What we wan to compare is **framework** performance -> with dynamic content not static

Can be ruby invoked in `/` ?

Reply:
I should mention that the previous version of Agoo considered an empty file a 404. The current does not so the `/` can be a static empty asset.

The app.rb explains some of this. I tried to make it clear.

Reply:
I understand, but all **frameworks** here invoke the `runtime` behind when calling `/`, it will not be a fair comparison, as the test protocol in to call `runtime`

Reply:
There is a comment in the app.rb file. By un-commenting the line that sets the route for `/` it will use Ruby. My apologies, I thought the inclusion of `/` was meant as a test for static assets. Apps usually have a mix so I thought it was normal.

Reply:
I wasn't sure what action I should take. Should I uncomment the offending line or were you planning to?

Reply:
> Should I uncomment the offending line or were you planning to?

Yes

> IMHO, it seems that servers with higher throughput are handicapped when calculating latency

Absolutely. That's way there is two tables :
+ latency
+ req/s

Reply:
Maybe I misunderstood. It looked like the latency was calculated on the same runs as the throughput which would mean latency was for different loads for each framework. If the latencies were calculated under the same load for each framework then my mistake. I did see the two table though.

Anyway, offending use of static assets has been disabled.

Reply:
All metrics are computed from the same runs. I have taken inspiration from https://github.com/TechEmpower/FrameworkBenchmarks

I'll have to explain this a bit more :stuck_out_tongue: now this process is quite stable, now